### PR TITLE
refactor: (ui-kit) change font size and line height px values to rem

### PR DESCRIPTION
**Pull request by V-Gira**
### Issue

- Fractional font-sizes are needed because of the introduction of a font size slider in the webapp.
- Because of px values passed as props in Team Setting, we need to fix the root font size at 16px or Team Setting would break when font sizes are changed at the browser lvl.

### Changes
- Slight change to the structure `Theme.tsx`, `GlobalCssVariables.tsx`and `Sizes.ts`(new) are moved to the `src/Theme` folder.
- Introduction of new `Sizes` theme variables, only `fontSizes` at the moment (eventually would include margins, paddings, etc)
- refactor of fontsize px values to rem theme variables
- refactor of lineheight values to rem

### CODE REVIEW

File: packages/react-ui-kit/src/Theme/Theme.tsx [hunk 1]
```diff
@@ -31,6 +33,7 @@ export enum THEME_ID {
 }
 
 export interface Theme extends ETheme {
+  fontSize: any;
```
[the remark this answers is not in the document]
Comment (V-Gira) [reply packages/react-ui-kit/src/Theme/Theme.tsx:36]: I knew I was forgetting something

File: packages/react-ui-kit/src/Theme/Sizes.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  xs: '0.6875rem',
+  s: '0.75rem',
+  m: '0.875rem',
+  l: '1rem',
+  xl: '1.125rem',
+  xxl: '1.25rem',
+  xxxl: '1.5rem',
+  xxxxl: '2rem',
+  xxxxxl: '2.5rem',
+  xxxxxxl: '3rem',
```
[the remark this answers is not in the document]
Comment (V-Gira) [reply packages/react-ui-kit/src/Theme/Sizes.ts:31]: Do you mean we only keep 5 or 6 variables with clearer names and we use the values directly for the rest?

[the remark this answers is not in the document]
Comment (V-Gira) [reply packages/react-ui-kit/src/Theme/Sizes.ts:31]: I'm going to use the same variables as webapp and remove the rest 
image

File: packages/react-ui-kit/src/Theme/Theme.tsx [hunk 3]
```diff
@@ -98,6 +108,7 @@ export interface Theme extends ETheme {
 
 export const themes: {[themeId in THEME_ID]: Theme} = {
   [THEME_ID.DEFAULT]: {
+    fontSizes: {...fontSizes},
```
[the remark this answers is not in the document]
Comment (V-Gira) [reply packages/react-ui-kit/src/Theme/Theme.tsx]: Nope, you're correct

**Review by atomrc — APPROVED** [review]

**Pull request merged**